src/components/Banners/SiteBanner.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import IconButton from "@leafygreen-ui/icon-button";
+import Icon from "@leafygreen-ui/icon";
+import Cookies from "js-cookie";
+import get from "lodash.get";
+import { GET_SITE_BANNER } from "gql/queries";
+import { SiteBannerQuery } from "gql/generated/types";
+
+const { yellow, blue, green, red } = uiColors;
+export const SiteBanner = () => {
+  const { data, loading } = useQuery<SiteBannerQuery>(GET_SITE_BANNER);
```

Comment:
I think it makes sense to poll this query. Eventually we will add an option to turn off polling in https://jira.mongodb.org/browse/EVG-12420 to accommodate users with bandwidth restrictions

Reply:
I do not think it needs to be polled. The banner wide notification is not something that we actively anticipate changes for. Also, new pages are opened often enough by clicking on a link or creating a patch that notifications will be seen

Reply:
Yeah i agree with Trey, Banners arent something that change to often. They are only ever displayed when an event that affects the evergreen service occurs.